pex/pex.py

```diff
@@ -580,7 +592,7 @@ def execute_interpreter(self):
         # type: () -> Any
 
         # A Python interpreter always inserts the CWD at the head of the sys.path.
-        sys.path.insert(0, ".")
+        sys.path.insert(0, "")
```

Comment:
Did "." never work? Or is "" just better spelling? I notice that in a vanilla python interpreter `sys.path[0]` is indeed `""`, so this definitely makes sense to me as a change. Just curious about why now.

Reply:
Just better spelling. At least consistent spelling with CPython and PyPy as you found out.